reference/5.1/Microsoft.PowerShell.Core/About/about_Calculated_Properties.md

```diff
@@ -0,0 +1,520 @@
+---
+Locale: en-US
+ms.date: 08/07/2020
+online version: https://docs.microsoft.com/powershell/module/microsoft.powershell.core/about/about_calculated_properties?view=powershell-5.1&WT.mc_id=ps-gethelp
+schema: 2.0.0
+title: about_Calculated_Properties
+---
+# About calculated properties
+
+## Short Description
+
+PowerShell provides the ability to dynamically add new properties and alter
+the formatting of objects output to the pipeline.
+
+## Long Description
+
+A number of PowerShell cmdlets transform, aggregate, or process input objects
+into output objects using parameters that allow the addition of new properties
+to those output objects. These parameters can be used to generate new,
+calculated properties on output objects based on the values of input objects.
+The calculated property is defined by attributes that specify the name of the
+new property, an expression to calculate the value, and optional formatting
+information. These attributes are supplied to the cmdlet parameters as a
+[hashtable](about_hash_tables.md) containing the attributes as key-value pairs.
```

Comment:
The only remaining issue I see here is the `attributes` concept. It introduces a new term to describe something otherwise familiar and collides with our other uses of the term "attribute". Rather than give a specific term for the hashtable fields that describe the calculated property, I feel like it's simpler just to say the parameter takes a hashtable to describe the new property.

Reply:
Ok. I have removed the use of "attributes"/